src/mmu/alloc.c

```diff
@@ -37,27 +37,41 @@ int liballoc_unlock() {
 
 void* liballoc_alloc(int number_of_pages) {
     uint64_t first_free_page = 0;
+    uint32_t counter = 0;
```

Comment:
Could we name this variable differently? `number_of_free_pages` maybe? WDYT?

Reply:
Changed to `free_pages_count`